CMakeLists.txt

```diff
@@ -1,5 +1,6 @@
 # CMake project definition file.
-cmake_minimum_required(VERSION 3.3...3.10)
+
+cmake_minimum_required(VERSION 3.8...4.0)
```

Comment:
Please don't bump CMake build to 3.8. Keep it 3.3 as minimum.

Reply:
Added cmake features shall not work with cmake older than 3.8:
https://cmake.org/cmake/help/latest/prop_gbl/CMAKE_CXX_KNOWN_FEATURES.html#prop_gbl:CMAKE_CXX_KNOWN_FEATURES
https://cmake.org/cmake/help/latest/prop_gbl/CMAKE_C_KNOWN_FEATURES.html#prop_gbl:CMAKE_C_KNOWN_FEATURES

Reply:
I have an alternative solution (hand-made) that works on older CMake versions normally: https://github.com/TheXTech/TheXTech/blob/main/cmake/build_props.cmake#L87-L103